L1Trigger/TrackTrigger/src/L1TrackQuality.cc

```diff
@@ -117,7 +117,17 @@ void L1TrackQuality::setL1TrackQuality(TTTrack<Ref_Phase2TrackerDigi_>& aTrack)
     aTrack.settrkMVA1(classification);
   }
 
-  if ((this->qualityAlgorithm_ == QualityAlgorithm::NN) || (this->qualityAlgorithm_ == QualityAlgorithm::GBDT)) {
+  else if (this->qualityAlgorithm_ == QualityAlgorithm::GBDT_cpp) {
```

Comment:
This is a copy of the runEmulatedTQ function essentially, I think we should either use the function or get rid of it

Reply:
This could be gotten rid of, though there might be situations where someone wants to test the BDT in isolation without the rest of the KF output emulation to check performance isn't dropping as a result of the BDT itself